### PR TITLE
Add style to the drop-down menu scroll bar

### LINKED ISSUE #3475: Add style to the drop-down menu scroll bar
#### Current Behavior
The scroll bar of the drop-down menu has a simple look

#### Desired Behavior
Make the drop-down menu look better by adding some style to the scroll bar

#### Screenshots / Mockups
layer5 scrollbar

#### Implementation

Use CSS to add some style to the scroll bar

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by tinniaru3005**
Signed-off-by: Arunima Chaudhuri <arunimachaudhuri2020@gmail.com>

**Description**
Add style to the drop-down menu scroll bar

This PR fixes #3475 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 66d8d0272b30bdf1d2c8531e32fdd6058d16147c at: https://638e3a97a0eae963fece5fb9--layer5.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thank you, @tinniaru3005. A couple of points of feedback:

1. Will you try using the same box-shadow from the desktop mega menu (it's a little more intense, offering stronger delineation of the menu from the page).
2. Will you try rounding the scrollbar so that it curves in the same way that the mobile menu curves with border-radius?

File: src/sections/General/Navigation/navigation.style.js [hunk 1]
```diff
@@ -374,6 +374,17 @@ const NavigationWrap = styled.header`
       max-height: 400px;
       overflow-y: scroll;
```
Comment (Nikhil-Ladha) [inline src/sections/General/Navigation/navigation.style.js:375]: You need to add `scrollbar-width: thin` to cover other browsers as well.

**Comment by l5io** [comment]
🚀 Preview for commit 7bd111d9d57b51b55d19844c0f99a87a6c2955f8 at: https://638f2fb6c41e6b17f0bd35c5--layer5.netlify.app

File: src/sections/General/Navigation/navigation.style.js (hunk 1 above)
Comment (tinniaru3005) [reply src/sections/General/Navigation/navigation.style.js:375]: @Nikhil-Ladha does it look good now?

**Comment by l5io** [comment]
🚀 Preview for commit ac050efea0fd8de547ee6330af4974e84909c52d at: https://638fb3f4c7956607ef65f89f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 84cba3f9f955dd086c11f5771018e28d3ebc357c at: https://6391eba862907317a82582ab--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
@leecalcote looks good to you?

**Comment by leecalcote** [comment]
Yes, indeed. I daresay that @tinniaru3005 did a darn fine job here. 👏

**Review by leecalcote — APPROVED** [review]

**Comment by tinniaru3005** [comment]
> Yes, indeed. I daresay that @tinniaru3005 did a darn fine job here. 👏

Thanks a lot, @leecalcote.

**Pull request merged**